addons/l10n_sa_edi/tests/compliance/standard/downpayment.xml

```diff
@@ -0,0 +1,235 @@
+<Invoice xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2">
```

Comment:
Better not to put this under compliance unless specifically asked for:
https://github.com/odoo/odoo/blob/feac6686202c3c2fc7643e1bfd90f444c58f53bc/addons/l10n_sa_edi/models/account_journal.py#L297

Reply:
Ok, moved the test file to a separate folder